go.sum

```diff
@@ -0,0 +1,16 @@
+github.com/mjibson/esc v0.2.0 h1:k96hdaR9Z+nMcnDwNrOvhdBqtjyMrbVyxLpsRCdP2mA=
```

Comment:
Let's exclude `go.sum` from the repo - it is frequently updated and we don't care.